src/igor/reservations.go

```diff
@@ -93,11 +93,14 @@ func (r *Reservations) Install(res *Reservation) error {
 		return nil
 	}
 
-	// pick a network segment
-	if v, err := r.NextVLAN(); err != nil {
-		return fmt.Errorf("error setting network isolation: %v", err)
-	} else {
-		res.Vlan = v
+	// Vlan wasn't specified by flag
+	if res.Vlan == 0 {
```

Comment:
If anyone has opinions about this line, I'd appreciate it. There may be a better way to indicate that igor needs to pick a VLAN.

Reply:
Where is this value getting set? I'm not seeing it...

Reply:
Go sets it to 0 by default, since it's an `int`. I assumed that if the `Vlan` field was 0, then the user must not have specified a VLAN with `-vlan`.  

I assumed that checking if it was zero would be OK because igor's configured VLANMin would probably always be greater than zero. That, and since VLAN zero is special, no one would ever try to choose it with `-vlan`.

Reply:
Gotcha. Yeah, given the circumstances, I don't think a little in-band signaling here will do us harm, though I share your discomfort. Given that igor is set to be overhauled, I don't think it is worth dwelling on the issue at this time. Let's move forward.